### PR TITLE
Rate Limiter Docs + Conv Assignee

### PR DESCRIPTION
**Pull request by kschiu**
Update docs for rate limiter + new PUT conversation assignee route
Screen Shot 2019-05-23 at 6 57 40 PM
Screen Shot 2019-05-23 at 6 42 20 PM

"Additional Rate Limiting"
Screen Shot 2019-05-23 at 6 36 49 PM

### CODE REVIEW

**Comment by kschiu** [comment]
@xdmnl  @travtechjones Could you take a quick pass when you have a chance? Thanks!

File: source/changelog.html.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ toc_footers:
 
 The changelog is the history of updates released. Front is committed in not breaking backwards compatibility between releases.
 
+## 2019-05-24 - Rate Limiter Updates
```
Comment (xdmnl) [inline source/changelog.html.md]: Please update the release date.

File: source/index.html.md [hunk 2]
```diff
@@ -87,6 +87,11 @@ Retry-After: 20
 
 When the rate limit is exceeded, the server will respond with a [**429 Too Many Requests**](https://tools.ietf.org/html/rfc6585#section-4) HTTP code with the header `Retry-After` to tell you how many seconds you need to wait before you can retry the request.
 
+### Additional Rate limiting
+Some resource intensive routes are subject to additional rate limiting to prevent strain on Front's infrastructure. Such routes have the header `X-Front-Tier`. If you are rate limited for such a route, you will receive the header `Retry-After`, even though `X-RateLimit-Remaining` may be greater than 0.
```
Comment (xdmnl) [inline source/index.html.md]: I think we should make it clear that:
- Both rate limits does not affect each other.
- This is a different limit that depends on the resources involved.
- This new rate limit is short lived and will only prevent sudden burst of requests.
- If they experience difficulties, they should contact api@frontapp.com

**Comment by kschiu** [comment]
Updated!
Screen Shot 2019-05-31 at 5 16 46 PM

**Pull request merged**